### PR TITLE
Initial commit of gmail profile

### PR DESCRIPTION
**Pull request by ChrisTruncer**
Hey Raffi,

This is my first profile, hopefully it should be fine.  Tested it out and I have beacon comms with it, and c2lint says it is clean.  If there's something else you need from me, just let me know.

### CODE REVIEW

File: normal/gmail.profile [hunk 1]
```diff
@@ -0,0 +1,83 @@
...
+	set L        "Mountain View";
+	set OU       "Google Mail";
+	set ST       "CA";
+	set validity "365";
+}
+
+http-get {
+	set uri "/mail/u/1/";
+	client {
+		header "Content-Type" "application/octet-stream";
```
Comment (rsmudge) [inline normal/gmail.profile]: No need for client to send a Content-Type unless it's a POST request. This stands out as somewhat strange.

File: normal/gmail.profile [hunk 2]
```diff
@@ -0,0 +1,83 @@
...
+	}
+
+	server {
+		header "X-Content-Type-Options" "nosniff";
+		header "X-Frame-Options" "SAMEORIGIN";
+		header "Cache-Control" "no-cache, no-store, max-age=0, must-revalidate";
+		header "X-XSS-Protection" "1; mode=block";
+		header "Server" "GSE";
+		output{
+			print;
```
Comment (rsmudge) [inline normal/gmail.profile]: It's up to you, but you can transform the output to look like something legitimate. Left as-is, you're just sending an encrypted blob back to the Beacon.

**Comment by ChrisTruncer** [comment]
ok, I'll look more into it.  I'll get back to this once updated.  You can close this if you want

File: normal/gmail.profile [hunk 3]
```diff
@@ -0,0 +1,83 @@
...
+		}
+	}
+}
+
+http-post {
+	set uri "/mail/u/0/";
+	client {
```
Comment (rsmudge) [inline normal/gmail.profile]: A Content-Type header from the client would be good as you're POSTing data back to the server. I recommend: application/x-www-form-urlencoded

File: normal/gmail.profile (hunk 1 above)
Comment (ChrisTruncer) [inline normal/gmail.profile]: yup, good point

**Comment by ChrisTruncer** [comment]
ok, thanks for the feedback, trying to learn by looking at your docs and example code here.  Appreciate the help, and I'll get this updated and resubmit later

**Pull request closed without merging**